include/deal.II/lac/utilities.h

```diff
@@ -50,6 +51,26 @@ namespace Utilities
                                       const VectorType &v0,
                                       const unsigned int k,
                                       VectorMemory<VectorType> &vector_memory);
+
+    /**
+     * Apply Chebyshev filter of @p op with a given positive @p degree
+     * to the input subspace @p x . @p a and @p b define the unwanted
+     * spectrum $ [a,b] $. @p vector_memory is used to allocate memory for
+     * temporary objects. Chebyshev polynomial will be scaled to unity at
+     * @p a_L to prevent overflow. If @p a_L equals to
+     * <code>numbers::signaling_nan<double>()</code>, no scaling is performed.
```

Comment:
Also, I think `signaling_nan<double>()` is the wrong thing here. We use that value for objects that we explicitly want to mark as uninitialized, so that *every* operation on these objects leads to an error. (That *should*, in principle, include just reading from the memory location, but in practice that is not how signaling NaNs are implemented.) What you want here is not to use an explicitly *uninitialized* value, but a special value outside the usual number range. The NaN from `numeric_limits` would be a useful value in that context.